tests/lib/rules/react-in-jsx-scope.js

```diff
@@ -44,9 +44,6 @@ ruleTester.run('react-in-jsx-scope', rule, {
     { code: 'var React; <x-gif />;' },
     { code: 'var React, App, a=1; <App attr={a} />;' },
     { code: 'var React, App, a=1; function elem() { return <App attr={a} />; }' },
-    {
-      code: 'var React, App; <App />;',
-    },
```

Comment:
Duplicate of:

https://github.com/jsx-eslint/eslint-plugin-react/blob/8e1a94b67d081fdc132e9a7e175db3fbf2e02956/tests/lib/rules/react-in-jsx-scope.js#L38